phase5/Solution1/ConsoleApp1/ReaderWriter.cs

```diff
@@ -0,0 +1,29 @@
+namespace ConsoleApp1;
+
+public class ReaderWriter
+{
+    public string ReadLine()
+    {
+        return Console.ReadLine();
+    }
+
+    private void WriteLine(string toWrite)
+    {
+        Console.WriteLine(toWrite);
+    }
+
+    public void writeList(List<string> toWrite)
+    {
+        foreach (var str in toWrite)
+        {
+            WriteLine(str);
+        }
+    }
+
+    public string PromptUserToEnterInfoAndReturn(String prompt)
+    {
+        WriteLine(prompt);
+        return ReadLine();
+    }
+    
```

Comment:
Using a `ReaderWriter` class is a neat idea; however using an Interface as a view or UI would be much better.
we want to show our final result to the user, here we use console to do so, but imagine that we have a graphical user interface, or a 3D environment. what should we do then? we should change our code totally!
**Brief idea:** using an interface for outputting the result
Nit: instead of passing a `list<string>` as an argument for `WriteList` method, you can use `IEnumberable` to be more generalized/